.gitignore

```diff
@@ -10,6 +10,8 @@
 # Ignore the default SQLite database.
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/database.yml
```

Comment:
That is only needed on my Coud9 setup... I didn't want to commit the file anywhere else.